### PR TITLE
{bio}[foss/2019b] Portcullis v1.2.2 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/804e6a96a58ba438ca331a1215335bea for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node123.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/20524cb32beb93a1848a1bc3251b7ceb for a full test report.

**Comment by verdurin** [comment]
@lexming Could you add the missing checksums, please?

**Comment by lexming** [comment]
@verdurin fixed :+1: new tests incoming

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node121.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/261eadf9560267773ef00c747e4c5414 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1e5e283463ea82c48d47c73d5911483e for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/312123986f84080b7b3475cc758c9541 for a full test report.

File: easybuild/easyconfigs/p/Portcullis/Portcullis-1.2.2-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+over splice sites. In order to address this, instead for creating a new RNAseq
+mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file
+generated by an RNAseq mapper of the user's own choice (e.g. Tophat2, Gsnap,
+STAR2 or HISAT2) as input (i.e. it's portable). It then, analyses and quantifies
+all splice junctions in the file before, filtering (culling) those which are
+unlikely to be genuine. Portcullis output's junctions in a variety of formats
+making it suitable for downstream analysis (such as differential splicing
+analysis and gene modelling) without additional work. Portcullis can also filter
+the original BAM file removing alignments associated with bad junctions."""
+github_account = 'maplesond'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/Portcullis/Portcullis-1.2.2-foss-2019b-Python-3.7.4.eb:20]: @lexming Looks generally fine but I'm not sure why this `github_account` is here given you don't refer to it elsewhere...

Comment (lexming) [reply easybuild/easyconfigs/p/Portcullis/Portcullis-1.2.2-foss-2019b-Python-3.7.4.eb:20]: @verdurin you are right, I was using `GITHUB_SOURCE` in an earlier version

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/464ff13bbf670d26d98176509a2805c5 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**